FWCore/Services/test/FpeTester.cc

```diff
@@ -54,6 +54,7 @@ namespace edmtest {
   }
 
   void
+  __attribute__((optnone))
```

Comment:
After looking carefully into Clang testsuite, I found `[[clang::optnone]]`, but I don't know how GCC (or other compilers) would handle that.